draft-ietf-uuidrev-rfc4122bis.md

```diff
@@ -1711,8 +1722,8 @@ Name-based UUIDs using UUIDv8:
 
 Advertising the Hash Algorithm:
 : Name-based UUIDs utilizing UUIDv8 do not allocate any available bits to identifying the hashing algorithm.
-  As such where common knowledge about the hashing algorithm for a given UUIDv8 name-space UUID is required, sharing the Hash Space ID proves useful for identifying the algorithm.
-  That is, to detail that SHA-256 was used to create a given UUIDv8 name-based UUID, an implementation may also share the "3fb32780-953c-4464-9cfd-e85dbbe9843d" hash space which uniquely identifies the SHA-256 hashing algorithm for the purpose of UUIDv8. Mind you that this needs not be the only method of sharing the hashing algorithm; this is one example of how two systems could share knowledge.
+  As such where common knowledge about the hashing algorithm for a given UUIDv8 name-space UUID is required, sharing the Hashspace ID proves useful for identifying the algorithm.
```

Comment:
is "Hashspace" a proper noun (not capitalized below)?

Reply:
s/name-space/namespace/ too?

Reply:
I didn't even see the "name-space" let me fix that and lowercase the hashspace.